phoebe/parameters/parameters.py

```diff
@@ -3800,8 +3801,8 @@ def calculate_residuals(self, model=None, dataset=None, component=None,
                 return residuals
 
     def calculate_chi2(self, model=None, dataset=None, component=None,
```

Comment:
I like that a lot. Let's go with that.

Reply:
I decided on a "hidden" `_calculate_mle` method since its redundant, would only add confusion to have another public method, and uses a magic string.

Please at least check the API docstrings and feel free to run on your cases to make sure it behaves as you'd expect (although I think the internal logic should be the same as your original commit).